wifiphisher/common/interfaces.py

```diff
@@ -243,6 +243,36 @@ def up_ifaces(self, ifaces):
         for i in ifaces:
             card = pyw.getcard(i.get_name())
             pyw.up(card)
+    
+    def _generate_random_address(self):
+        """
+        Make and return the randomized MAC address
+
+        :param self: A NetworkManager object
+        :type self: NetworkManager
+        :return: A MAC address
+        :rtype str
+        """
+
+        mac_addr = "02:00:00:{:02x}:{:02x}:{:02x}".format(random.randint(0, 255),\
```

Comment:
I'm wondering if it would be possible to change the range to guarantee the OUI so instead of (0, 255) to (10,50) for instance.